gateway/1.5/modules/ROOT/pages/policies-included-route-weight.adoc

```diff
@@ -0,0 +1,143 @@
+= Traffic Management Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:keywords: route weight, traffic management
+:page-aliases: policies::policies-included-tls.adoc
+
+[%autowidth.spread,cols="a,a"]
+|===
+>s| Policy Name | Traffic Management
+>s|Summary      | Manages API traffic to multiple uspstream
+>s|Category | Security
+>s| First Flex Gateway version available | v1.4.0
```

Comment:
Is traffic management functionality first available in 1.5.0, and is the standard TLS functionality still first available in 1.4.0?